### ISSUE TITLE
Deleting an icon forces the other icons shift.

### ISSUE THREAD

**Issue by genaDeus** [incognito1234/KeePass-Custom-Icon-Dashboarder#8]
Deleting an icon with "Remove" action shift/reorder all the icons in the base. Note, if to delete an icon with the standart feature "Edit entry"->Icon->"Icon Picker" all works just fine.

**Comment by amadeogarcia** [comment]
I also have this issue.

**Comment by incognito1234** [comment]
Hi,
I was unable to reproduce this issue. Could you give the version of KeePass you use and give a step-by-step plan to reproduce it ?
Perhaps the last version of the plugin will solve this issue.
Thanks for your feedback,
Jareth

**Comment by amadeogarcia** [comment]
I switched to KeePassXC a long time ago, but I just tried this again (with latest versions of both KeePass and the plugin, i.e. v2.53.1 and v1.3 respectively, as I didn't have them installed anymore) and I'm not sure what the issue was. Sure, it kind of makes all the other icons "shift" as it updates the list after removing one icon, which makes it seem a bit laggy, but that's just how the plugin behaves, even when opening it. Perhaps we could mark this as a performance issue?

**Edit:** sorry, I forgot the step-by-step. IIRC, and if not then what I just did is wrong, you just go to 'Tools -> Custom Icon Dashboard', select an icon and choose the "Remove" action.
